routes/weddingRouter.js

```diff
@@ -4,11 +4,19 @@ const Wedding = require("../models/weddings");
 const guestsRouter = require("./guestsRouter");
 const vendorRouter = require("./vendorRouter");
 
+const announcementsRouter = require("./announcementRouter");
+const registryRouter = require("./registryRouter");
+
 const express = require("express");
 
 const router = express.Router();
+
 router.use("/:weddingId/guests", guestsRouter);
 router.use("/:weddingId/vendors", vendorRouter);
+
+router.use("/:weddingId/announcements", announcementsRouter);
+router.use("/:weddingId/registry", registryRouter);
```

Comment:
To keep it consistent with the rest of the endpoints, I'd suggest pluralizing "registry":

```suggestion
router.use("/:weddingId/registries", registryRouter);
```

Reply:
fixed/outdated